### PR TITLE
SRE-2537: add license as expected by OpenSSF Scorecard

### PR DESCRIPTION
**Pull request by grom72**
Inherit BSD-2-Clause-Patent License
from the main (daos-stack/daos) project.

### CODE REVIEW

**Review by osalyk — DISMISSED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (c) 2019-2024 Intel Corporation
```
Comment (janekmi) [inline Makefile:2]:
```suggestion
# Copyright (c) 2019-2024 Intel Corporation

```

**Review by janekmi — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (grom72) [reply Makefile:2]: Done

**Review by janekmi — DISMISSED** [review]

**Review by osalyk — DISMISSED** [review]

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
Pity that 58600dc67f2d28233b2b40001eff570561e95bed was not pushed to 5e275055a78de3de01d146e44a544256c354be47 as we usually do to overwrite the read-only notice.  Please add a revert of 20039197213d3026c494f5e38fa87fce2a81eab5 to this PR to remove that erroneous message.

**Comment by grom72** [comment]
> Pity that [58600dc](https://github.com/daos-stack/fuse3/commit/58600dc67f2d28233b2b40001eff570561e95bed) was not pushed to [5e27505](https://github.com/daos-stack/fuse3/commit/5e275055a78de3de01d146e44a544256c354be47) as we usually do to overwrite the read-only notice. Please add a revert of [2003919](https://github.com/daos-stack/fuse3/commit/20039197213d3026c494f5e38fa87fce2a81eab5) to this PR to remove that erroneous message.

Done

**Review by brianjmurrell — APPROVED** [review]

**Review by brianjmurrell — APPROVED** [review]

**Pull request merged**